src/AboutUser/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import { Link, withRouter } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { logoutUser } from '../actions/authentication';
+
+
+
+class AboutUser extends Component{
+  onLogout(e) {
+    e.preventDefault();
+    this.props.logoutUser(this.props.history);
+  }
+  render() {
+        const { user} = this.props.auth;
+  return(
+      <div>
+          <h1>HI, Here some information about you</h1>
+          <h1>It is your name</h1>
+           {user.name}
```

Comment:
should it be in a selector? p or h4 or span?

Reply:
it was just for example ,that server side working